src/Maker/MakeResetPassword.php

```diff
@@ -320,4 +317,84 @@ private function successMessage(InputInterface $input, ConsoleStyle $io, string
         $io->text('Then open your browser, go to "/reset-password" and enjoy!');
         $io->newLine();
     }
+
+    private function generateRequestEntity(Generator $generator, ClassNameDetails $requestClassNameDetails, ClassNameDetails $repositoryClassNameDetails, string $userClass): void
+    {
+        $entityClassGenerator = new EntityClassGenerator($generator, $this->doctrineHelper);
+
+        $requestEntityPath = $entityClassGenerator->generateEntityClass($requestClassNameDetails, false);
+
+        $generator->writeChanges();
+
+        $manipulator = new ClassSourceManipulator(
+            $this->fileManager->getFileContents($requestEntityPath)
+        );
+
+        $manipulator->addInterface(ResetPasswordRequestInterface::class);
+
+        $manipulator->addTrait(ResetPasswordRequestTrait::class);
+
+        $manipulator->addConstructor([
+            (new Param('user'))->setType('object')->getNode(),
+            (new Param('expiresAt'))->setType('\DateTimeInterface')->getNode(),
+            (new Param('selector'))->setType('string')->getNode(),
+            (new Param('hashedToken'))->setType('string')->getNode(),
+        ], <<<'CODE'
+<?php
+$this->user = $user;
+$this->initialize($expiresAt, $selector, $hashedToken);
+CODE
+        );
+
+        $manipulator->addManyToOneRelation((new RelationManyToOne())
+            ->setPropertyName('user')
+            ->setTargetClassName("App\Entity\User") // customization fail in php 7.3
```

Comment:
What's the problem in PHP 7.3? We need to be explicit. And if there IS a problem currently in 7.3, then that's a block for the PR and we should be clear that this is still a "TODO" (like by posting a comment about it in the PR or in the PR description)

Reply:
Not php 7.3, i wrote that because only the test for php 7.3 failed in travis.
https://github.com/symfony/maker-bundle/blob/bea8c3c959e48a2c952cc7c4f4f32964be8b8874/tests/Maker/MakeResetPasswordTest.php#L173

Reply:
jrushlow added new code. Our targetClass is generaated like a fetch class `UserCustom::class`  not in FQCN `"App\Entity\UserCustom"`